snuba/datasets/processors/spans_processor.py

```diff
@@ -100,12 +100,14 @@ def _process_span_event(
             processed["group_raw"] = 0
 
         # timestamps
-        processed["start_timestamp"], processed["start_ms"] = self.__extract_timestamp(
-            span_event["start_timestamp_ms"] / 1000,
-        )
-        processed["end_timestamp"], processed["end_ms"] = self.__extract_timestamp(
-            (span_event["start_timestamp_ms"] + span_event["duration_ms"]) / 1000,
-        )
+        start_timestamp_sec = span_event["start_timestamp_ms"] / 1000
+        processed["start_timestamp"] = int(start_timestamp_sec)
+        _, processed["start_ms"] = self.__extract_timestamp(start_timestamp_sec)
```

Comment:
Mind modifying `__extract_timetamp()` to return int instead of datetime if that's what we need now? (these are the only 2 places it's used)